src/Components/Config/configHooks.tsx

```diff
@@ -157,14 +166,21 @@ function transformItem(item: unknown): any {
   return config;
 }
 
-function transformConfigResponse(configResponse: ConfigApiResponse[]): Config {
-  const output: Config = {};
+function transformConfigData(configData: ConfigApiResponse[]): Config {
+  const transformedConfig: Config = {};
+
+  configData.forEach((item) => {
+    const { name, data } = item;
+    let configOptions = data;
+
+    if (typeof configOptions === 'string' && isValidJson(configOptions)) {
+      configOptions = JSON.parse(configOptions);
+    }
```

Comment:
I don't think that we need this.

Reply:
This is needed to parse the json strings sent from the backend.

Reply:
It works for me without it.

Reply:
Languages do not work properly without it.

![Screenshot 2024-06-03 at 11 57 53 AM](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/f57b2401-e2d3-4e05-a806-1361568a58eb)

Reply:
It works for me (I removed the other languages to test that it would change if I made an update).
![image](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/62856626/baa49874-a282-4f4c-9c67-d499c62e2a77)

Reply:
Maybe try removing your current config and importing it again?

Reply:
Re-importing the config did it for me. Pushed the updated code.